server/app/views/admin/AdminLayoutBaseView.java

```diff
@@ -11,20 +13,29 @@
 import views.admin.shared.AdminCommonHeader;
 
 /**
- * Admin base view class used to render the supplied Thymeleaf page template. This view is tied to
- * layout template file {@code AdminLayout.html}.
+ * {@link AdminLayoutBaseView} is used to render the supplied Thymeleaf page template. This view is
+ * tied to layout template {@link LayoutTemplate#ADMIN_LAYOUT}.
  *
  * @param <TModel> A class or record that implements {@link BaseViewModel}
  */
-public abstract class AdminBaseView<TModel extends BaseViewModel> extends BaseView<TModel> {
-  public AdminBaseView(
+public abstract class AdminLayoutBaseView<TModel extends BaseViewModel> extends BaseView<TModel> {
```

Comment:
Renaming to align with naming conventions that shook out while [writing docs](https://github.com/civiform/civiform/wiki/Working-with-Admin-Thymeleaf-Pages#naming-convention-2).

- [Prefix]LayoutBaseView.java
- [Prefix]Layout.html